hw/hw_05_task_01.py

```diff
@@ -0,0 +1,131 @@
+"""
+Необходимо создать 3 класса и взаимосвязь между ними (Student, Teacher, Homework).
+
+Наследование в этой задаче использовать не нужно.
+Для работы с временем использовать модуль datetime
+1. Homework принимает на вход 2 атрибута: текст задания и количество дней
+на это задание
+Атрибуты:
+    text - текст задания
+    deadline - хранит объект datetime.timedelta с количеством
+    дней на выполнение
+    created - c точной датой и временем создания
+Методы:
+    is_active - проверяет не истело ли время на выполнение задания,
+    возвращает boolean
+2. Student
+Атрибуты:
+    last_name
+    first_name
+Методы:
+    do_homework - принимает объект Homework и возвращает его же,
+    если задание уже просрочено, то печатет 'You are late' и возвращает None
+3. Teacher
+Атрибуты:
+     last_name
+     first_name
+Методы:
+    create_homework - текст задания и количество дней на это задание,
+    возвращает экземпляр Homework
+    Обратите внимание, что для работы этого метода не требуется сам объект.
+PEP8 соблюдать строго.
+Всем перечисленным выше атрибутам и методам классов сохранить названия.
+К названием остальных переменных, классов и тд. подходить ответственно -
+давать логичные подходящие имена.
+"""
+from datetime import datetime, timedelta
+
+
+class Homework:
+    """
+    A class to represent a homework.
+
+    Attributes
+    ----------
+    text : str
+        text of homework
+    deadline: int
+        num of days to do homework
+    created: datetime.timedelta
+        date and time of deadline
+
+    Methods
```

Comment:
You have documentation here and in the docstring.  This is duplication. It is better to remove it from here.